org/party.go

```diff
@@ -122,14 +122,20 @@ type Website struct {
 // The definition found here is based on the details required for spain.
 // If your country requires additional fields, please let us know.
 type Registration struct {
-	UUID    *uuid.UUID `json:"uuid,omitempty" jsonschema:"title=UUID"`
-	Office  string     `json:"office,omitempty" jsonschema:"title=Office"`
-	Book    string     `json:"book,omitempty" jsonschema:"title=Book"`
-	Volume  string     `json:"volume,omitempty" jsonschema:"title=Volume"`
-	Sheet   string     `json:"sheet,omitempty" jsonschema:"title=Sheet"`
-	Section string     `json:"section,omitempty" jsonschema:"title=Section"`
-	Page    string     `json:"page,omitempty" jsonschema:"title=Page"`
-	Entry   string     `json:"entry,omitempty" jsonschema:"title=Entry"`
+	UUID *uuid.UUID `json:"uuid,omitempty" jsonschema:"title=UUID"`
+	// Identifier for the registration.
+	Label string `json:"label,omitempty" jsonschema:"title=Label"`
+	// Actual Code or ID that identifies the registration.
+	Code    string `json:"code,omitempty" jsonschema:"title=Code"`
+	Office  string `json:"office,omitempty" jsonschema:"title=Office"`
+	Book    string `json:"book,omitempty" jsonschema:"title=Book"`
+	Volume  string `json:"volume,omitempty" jsonschema:"title=Volume"`
+	Sheet   string `json:"sheet,omitempty" jsonschema:"title=Sheet"`
+	Section string `json:"section,omitempty" jsonschema:"title=Section"`
+	Page    string `json:"page,omitempty" jsonschema:"title=Page"`
+	Entry   string `json:"entry,omitempty" jsonschema:"title=Entry"`
+	// Any additional useful data.
+	Meta cbc.Meta `json:"meta,omitempty" jsonschema:"title=Meta"`
```

Comment:
Meta feels out of place here... meta is meant for end-users who need to communicate additional information between themselves and their customers in a given context, I don't foresee a use-case for the registration details which is very much a thing for governments as opposed to customers.